src/main.rs

```diff
@@ -145,7 +170,7 @@ fn analyze_package(package: PathBuf,
     let start_time = Instant::now();
 
     // APKTool app decompression
-    decompress(config, &package);
+    decompress(config, &package)?;
```

Comment:
Same here, a `chain_err()` saying that the "decompression of the package failed" would be great :)